core/modules/system/tests/system.test

```diff
@@ -2360,6 +2360,35 @@ class SystemValidTokenTest extends BackdropWebTestCase {
 }
 
 /**
+ * Tests backdrop_set_message() and related functions.
+ */
+class BackdropSetMessageTest extends BackdropWebTestCase {
+
+  public static function getInfo() {
+    return array(
+      'name' => 'Messages',
+      'description' => 'Tests that messages can be displayed using backdrop_set_message().',
+      'group' => 'System',
+    );
+  }
```

Comment:
In Backdrop, we don't have `getInfo()` methods in the test classes. Instead move this information into `system.tests.info` to register the test.